pkg/util/globalerror/errors.go

```diff
@@ -55,6 +55,7 @@ const (
 
 	SampleTimestampTooOld    ID = "sample-timestamp-too-old"
 	SampleOutOfOrder         ID = "sample-out-of-order"
+	SampleTooOld             ID = "sample-too-old"
```

Comment:
We already have `SampleTimestampTooOld`. Why don't we reuse it? I think from a UX perspective it's quite confusing having "sample-timestamp-too-old" and "sample-too-old". I would just reuse "SampleTimestampTooOld" but with a different error message, and then in the runbook we cover both cases (OOO is enabled or not).